.env

```diff
@@ -0,0 +1 @@
+SECRET='IOaheihEOHaAOIEHaiohIOEHaioh'
```

Comment:
**nitpick:** Remove this file at https://github.com/TirusTimes/tirus-times-back/pull/2/commits/b98c4a4538fcba3487999b027ea61abe4739074f and add that at `.env.example`